sdk/eventhub/azure-eventhubs/azure/eventhub/_client_base.py

```diff
@@ -104,15 +104,15 @@ def _build_uri(address, entity):
 
 
 class ClientBase(object):  # pylint:disable=too-many-instance-attributes
-    def __init__(self, host, event_hub_path, credential, **kwargs):
-        self.eh_name = event_hub_path
-        path = "/" + event_hub_path if event_hub_path else ""
-        self._address = _Address(hostname=host, path=path)
+    def __init__(self, fully_qualified_namespace, eventhub_name, credential, **kwargs):
+        self.eventhub_name = eventhub_name
+        path = "/" + eventhub_name if eventhub_name else ""
```

Comment:
Are there sane situations where eventhub_name can be None? (addressing the ternary here as opposed to failing fast)

Reply:
No `eventhub_name` cannot be None - so if we are allowing it elsewhere, that should be fixed :)